src/Reflectivity-VariableBreakpoint-Extensions/ClyBreakOnVarAccessCommand.class.st

```diff
@@ -0,0 +1,25 @@
+"
+I break on all accesses to selected target variable in the source code
+"
+Class {
+	#name : #ClyBreakOnVarAccessCommand,
```

Comment:
This is very confusing names: ClyBreakOnAccessCommand vs ClyBreakOnVarAccessCommand. And same for subclasses and superclasses.
What the difference between them? 
I am sure we can avoid these parallel hierarchies.

Reply:
I did not look deeply in the code. But my first idea to solve this is to ask variables to install breakpoints instead of doing it directly in the command

Reply:
For example check other SycVariableCommand's. All of them are available in variables list and in source code (when variable name is selected).

Reply:
The problem (to me) is that it is very hard to understand and build commands with all that subclassing. Maybe I got something wrong and it can be simplified.

Difference is:
- ClyBreakOnAccessCommand => takes a class (or an object) and break on *all* variable acccesses
- ClyBreakOnVarAccessCommand => takes a particular variable

Also if I understood right, it is needed to have three hierarchies because they don't apply to the same contexts. I found no way of having the menus right in those three places (source code, on the class and on the variable view).

Reply:
Hm. I see following subclasses:
- ClyBreakOnAccessCommand <-- ClyBreakOnVarCommand <-- SycVariableCommand - which are commands for the list of variables
- ClyBreakOnVarAccessCommand <-- ClyVariableBreakpointCommand <-- ClyDebuggingCommand <-- SycSourceCodeCommand - which are commands for the given astNode in given method

I have impression that PR/commit is corrupted. I found two files ClyVariableBreakpointCommand.class.st in different packages (not extension). Also there is  SycVariableBreakpointCommand. 
It looks like you did some repackaging and rename and then Iceberg commit became wrong.

Reply:
Indeed I renamed a package. I see it also make other tests failing related to the manifest.
I will look at that tomorrow morning, this is strange :/

Reply:
This is non-blocking, and I cannot afford more time right now to work on it. Could we do an issue to improve the command hierarchy and work from there so that it does not block the PR?

Reply:
Yes. I am always for integrate/improve cycle.